PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskPID.cxx

```diff
@@ -1277,23 +1277,23 @@ void AliAnalysisTaskPID::UserCreateOutputObjects()
     fTPCclusterStudies = new TObjArray*[arrNumbers];
     const Int_t histNumbers = 3;
     
-    const Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
+    constexpr const Int_t nBinsY[histNumbers+1] = {8, 4, 11, 15};
```

Comment:
Please remove the unnecessary "const" (only constexpr is enough)

Reply:
Done